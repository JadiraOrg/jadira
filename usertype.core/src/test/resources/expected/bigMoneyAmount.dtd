<!ELEMENT dataset (
    BIGMONEYAMOUNT*)>

<!ELEMENT BIGMONEYAMOUNT EMPTY>
<!ATTLIST BIGMONEYAMOUNT
    ID CDATA #REQUIRED
    BIGMONEY CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

