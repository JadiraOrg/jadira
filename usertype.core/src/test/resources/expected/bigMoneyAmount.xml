<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "bigMoneyAmount.dtd">
<dataset>
  <BIGMONEYAMOUNT ID="0" BIGMONEY="100.00" NAME="test_0"/>
  <BIGMONEYAMOUNT ID="1" BIGMONEY="100.10" NAME="test_1"/>
  <BIGMONEYAMOUNT ID="2" BIGMONEY="0.99" NAME="test_2"/>
  <BIGMONEYAMOUNT ID="3" NAME="test_3"/>
</dataset>